### DISCUSSION TITLE
Select / Combo-box with multi-select?

### DISCUSSION THREAD

**Discussion by dajones537** [uswds/uswds#5600] [Q&A] [answered]
I want to do something like this:
![image](https://github.com/uswds/uswds/assets/123194829/037edfc1-630f-44fa-9ce9-91d7f190d222)
![image](https://github.com/uswds/uswds/assets/123194829/206ec3ea-90df-485b-9f0e-09b65cb35551)
So basically a select input where you can select multiple, and when you do they appear as boxes you can delete. Perhaps also you could type to filter. There is a select multiple in the pattern lab, but it's not like this at all. This is what the pattern lab multi select looks like:
![image](https://github.com/uswds/uswds/assets/123194829/2b1dd02f-534e-4e5b-9dcb-5d4baf8e2ea3)
I did find this here:
![image](https://github.com/uswds/uswds/assets/123194829/94d78542-73f1-42ff-afaa-bd635be0d9de)
But this seems to allow just one selection.

**Comment by amyleadem** [comment, marked as answer]
Hello @dajones537,
Thanks for adding this idea to our discussion board. We have an open feature request for this functionality in issue https://github.com/uswds/uswds/issues/4863

It would be great if you could add an up-vote and/or comment to that issue so that we can better understand community interest in the feature. This also would help us consolidate discussion about the feature request in one spot. Let me know if you have any questions!

**Reply by dajones537** [reply]
Cool deal, thanks. In the meantime, I implemented a multi-select of my own in order to match our project's wireframes.

**Comment by ryparker** [comment]
For anyone looking for the current status: USWDS 3.14.0's combo box is still single-select. Adding `multiple` to its underlying select does not turn the enhanced component into a supported multi-select, and USWDS does not currently ship a chip-based multi-select component.

The earlier [multi-select request, #4863](https://github.com/uswds/uswds/issues/4863), was closed without implementing it.

For a manageable set of choices, [checkboxes](https://designsystem.digital.gov/components/checkbox/) provide the built-in multiple-selection option. A searchable multi-select for a large set of choices would need a separate implementation, including its own keyboard, focus, and screen-reader testing. It is not something enabled by a combo-box setting.
